apps/webapp/components/layout/footer/faq.tsx

```diff
@@ -19,10 +19,10 @@ export function FAQ({ lang, dict }: FAQProps) {
           {dict.faq.questions.map(
             (item: { question: string; answer: string }, index: number) => (
               <AccordionItem key={index} value={`item-${index}`}>
-                <AccordionTrigger className="transition-color flex w-full items-center justify-between px-6 py-4 text-left font-medium focus:outline-none data-[state=open]:bg-card/50">
+                <AccordionTrigger className="flex w-full items-center justify-between rounded-b px-6 py-4 text-left font-sans text-base font-medium focus:outline-none data-[state=open]:bg-muted data-[state=open]:opacity-70">
```

Comment:
Can we add the class `data-[state=open]:decoration-none` (`text-decoration: none;`)? So when is active, on mobile won't get stuck with the underline (because remains on active link state)